src/nodes/Network/IPFSSubscribe.js

```diff
@@ -17,46 +16,44 @@ IPFSSub.title = "IPFSSubscribe";
 
 IPFSSub.prototype.onReceiveMsg = async function(msg) {
   console.log(msg)
-  this.message = msg.data.toString()
+  this.message = Buffer.from(msg.data).toString('utf-8')
```

Comment:
Have to decode uint8array here, otherwise the message are comma separated numbers